uwp/launch-resume/launch-screen-snipping.md

```diff
@@ -20,24 +20,27 @@ The **ms-screenclip:** URI allows your app to automatically open up and start a
 | Parameter | Type | Required | Description |
 | --- | --- | --- | --- |
 | source | string | no | A freeform string to indicate the source that launched the URI. |
+| type | string | no | A string value to indicate which special type of capture is requested. This parameter can be omitted when starting a new snip. Values supported include: snapshot, recording* |
```

Comment:
Probably best to escape the asterisk at the end of this line to prevent any future unintentional formatting.